asciinema/config.py

```diff
@@ -28,12 +28,12 @@ def api_url(self):
     @property
     def api_token(self):
         try:
-            return self.config.get('api', 'token')
+            return self.env.get('ASCIINEMA_API_TOKEN', self.config.get('api', 'token'))
```

Comment:
I think this will raise exception if you set env var but don't have `api.token` in config file because `self.config.get('api', 'token')` will get evaluated first, before its value can be passed to `self.env.get(..)`.
Maybe using Python's `or` would do the job here?